src/js/tabs/firmware_flasher.js

```diff
@@ -119,132 +138,195 @@ TABS.firmware_flasher.initialize = function (callback) {
                         if($.inArray(target, selectTargets) == -1) {
                             selectTargets.push(target);
                             var select_e =
-                                    $("<option value='{0}'>{0}</option>".format(
-                                            descriptor.target
-                                    )).data('summary', descriptor);
+                                $("<option value='{0}'>{0}</option>".format(
+                                        descriptor.target
+                                ));
                             boards_e.append(select_e);
                         }
                     });
                 });
 
             TABS.firmware_flasher.releases = builds;
 
-            chrome.storage.local.get('selected_board', function (result) {
+            ConfigStorage.get('selected_board', function (result) {
                 if (result.selected_board) {
                     var boardBuilds = builds[result.selected_board]
                     $('select[name="board"]').val(boardBuilds ? result.selected_board : 0).trigger('change');
                 }
             });
         }
 
-        function buildBoardOptions(releaseData, showDevReleases) {
-            if (!releaseData) {
-                $('select[name="board"]').empty().append('<option value="0">Offline</option>');
-                $('select[name="firmware_version"]').empty().append('<option value="0">Offline</option>');
-            } else {
-                var boards_e = $('select[name="board"]');
-                var versions_e = $('select[name="firmware_version"]');
-
-                var releases = {};
-                var sortedTargets = [];
-                var unsortedTargets = [];
-                releaseData.forEach(function(release){
-                    release.assets.forEach(function(asset){
-                        var targetFromFilenameExpression = /betaflight_([\d.]+)?_?(\w+)(\-.*)?\.(.*)/;
-                        var match = targetFromFilenameExpression.exec(asset.name);
-
-                        if ((!showDevReleases && release.prerelease) || !match) {
-                            return;
-                        }
-                        var target = match[2];
-                        if($.inArray(target, unsortedTargets) == -1) {
-                            unsortedTargets.push(target);
-                        }
-                    });
-                    sortedTargets = unsortedTargets.sort();
+        function processBoardOptions(releaseData, showDevReleases) {
+            var releases = {};
+            var sortedTargets = [];
+            var unsortedTargets = [];
+            releaseData.forEach(function(release){
+                release.assets.forEach(function(asset){
```

Comment:
Please, space between `)` and `{` at both `forEach`. I know most of them are from older code, but now that we modify them we can get rid of it ;)